### PR TITLE
Specify full ops routes in plug

### PR DESCRIPTION
**Pull request by Stratus3D**
[Card](https://rentpath.atlassian.net/browse/SRV-4883)

* Specify full ops routes in plug
* Remove `Plug.Router` from plug

### CODE REVIEW

**Review by pasha — APPROVED** [review]

File: lib/plug/ex_ops_plug.ex [hunk 1]
```diff
@@ -2,28 +2,19 @@ defmodule ExOps.Plug do
...
   """
-  use Plug.Router
-
   import Plug.Conn
 
-  plug(:match)
-  plug(:dispatch)
-
-  get "/heartbeat" do
+  @spec call(Plug.Conn.t(), list()) :: Plug.Conn.t()
```
Comment (KevinMarth) [inline lib/plug/ex_ops_plug.ex:7]: Should this module implement the `Plug` behavior at https://hexdocs.pm/plug/Plug.html?

Comment (Stratus3D) [reply lib/plug/ex_ops_plug.ex:14]: Yes! Not sure why the docs I was looking at didn't show it as a behavior. PR updated.

Comment (KevinMarth) [reply lib/plug/ex_ops_plug.ex:14]: See, for example, https://github.com/rentpath/mobile_gateway/blob/master/apps/web/lib/web/plugs/session_id.ex

Comment (KevinMarth) [reply lib/plug/ex_ops_plug.ex:14]: `@impl` can be in preference to `@spec` here.

**Review by KevinMarth — APPROVED** [review]

**Pull request merged**